gsutil.py

```diff
@@ -74,7 +74,6 @@ def OutputAndExit(message):
     ('funcsigs', ''),  # mock dependency
     ('google-reauth-python', ''),  # Package name: google_reauth
     ('pyu2f', ''),  # google_reauth dependency
-    ('oauth2client', ''),
```

Comment:
Why did you move this out of `third_party` and move it into vendored? You could just change the location of the original submodule and keep everything else set up in the same way.

Reply:
This was done for PyPI. Because we are not maintaining our own version of oauth2client and not relying on the PyPI version, it was moved from here to `vendored_lib_dirs` in  `gslib.__init__.py` When gsutil gets installed via PyPI and invoked directly using `gslib.__main__.py` we might end up including the incorrect version.  The note on Line 91 explains this to some extent.

Reply:
To give some context, it is the same reason why boto is not included here, and is present under the `vendored_lib_dirs` section.